nginx/README.md

```diff
@@ -277,13 +306,6 @@ metadata:
       ]
   labels:
     name: nginx
-spec:
```

Comment:
Why remove this?

Reply:
Because that is actually part of the configuration explained above, and confusing if we keep it here as well